scripts/Inelastic/IndirectImport.py

```diff
@@ -80,7 +81,7 @@ def is_supported_f2py_platform():
     @returns True if we are currently on a platform that supports the F2Py
     libraries, else False.
     """
-    if (_os_env().startswith("Windows")
+    if (_os_env().startswith("Windows") and "CONDA_PREFIX" not in os.environ
```

Comment:
Can I just double check this? I thought one of the benefits of the Conda approach was that we would be able to support f2py across all the platforms. This seems to exclude it though for any Conda build

Reply:
That will be the intention, but I was in two minds of whether to include it in this PR so for the purposes of testing I turned it off. I added the f2py modules through the PR https://github.com/mantidproject/mantid/pull/31862 . There's still some thought required on how to integrating these libraries, e.g. adding the relevant things to the python search path. 

As I understand it, the libraries used in `IndirectImport` assume that we'll use the libraries in that directory, e.g `QLdata_win64.cp38-win_amd64.pyd`. These libraries are linked against a certain gfortran math library, which is shipped in our third-party dependencies https://github.com/mantidproject/thirdparty-msvc2015/tree/master/bin/mingw . Without these dependencies (as we don't download the third-party repo), the libraries have unmet dependencies. Therefore, the current setup won't work with conda, so a bit more thought needs to go into it.

Admittedly this isn't a great solution, but hopefully it will only be a temporary change, while we plan our f2py strategy going forward.